controllers/utils_test.go

```diff
@@ -39,7 +39,7 @@ var _ = Describe("Controller Suite", func() {
 			}
 			serviceToStatefulSpec = map[string]*v1alpha1.CDAPStatefulServiceSpec{
 				serviceLogs:          &master.Spec.Logs.CDAPStatefulServiceSpec,
-				serviceAppFabric:     &master.Spec.AppFabric.CDAPStatefulServiceSpec,
+				serviceAppFabric:     nil,
```

Comment:
Do we need to include appfabricProcessorService here?

Reply:
Yes, added `serviceAppFabricProcessor`